indicator.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2023 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package cmw
+
+type Indicator uint
+
+const (
+	ReferenceValues = 1 << iota
+	Endorsements
+	Evidence
+	AttestationResults
+	TrustAnchors
+)
+
+const IndicatorNone = 0
```

Comment:
it would be more clear if the IndicatorNone would be integrated with the other const of the message bit map, starting from zero.

Reply:
we can't, the use of `iota` is not compatible with that.